data-canary/scripts/creaturescripts/login.lua

```diff
@@ -97,7 +97,16 @@ function login.onLogin(player)
 	-- Set Client XP Gain Rate --
 	if configManager.getBoolean(configKeys.XP_DISPLAY_MODE) then
 		local baseRate = player:getFinalBaseRateExperience()
-		player:setBaseXpGain(baseRate * 100)
+		baseRate = baseRate * 100
+		if configManager.getBoolean(configKeys.VIP_SYSTEM_ENABLED) then
+			local vipBonusExp = configManager.getNumber(configKeys.VIP_BONUS_EXP)
+			if (vipBonusExp > 0 and player:isPremium()) then
+				vipBonusExp = (vipBonusExp > 100 and 100) or vipBonusExp
+				baseRate = baseRate * (1 + (vipBonusExp / 100))
+				player:sendTextMessage(MESSAGE_BOOSTED_CREATURE, "Normal base xp is: " .. baseRate .. " because you VIP bonus of " ..vipBonusExp)
```

Comment:
change: `"Normal base xp is: " .. baseRate .. " because you VIP bonus of " ..vipBonusExp`
by: `"Normal base xp is: " .. baseRate .. "%, because you are VIP, bonus of " .. vipBonusExp`